src/config.rs

```diff
@@ -97,6 +97,7 @@ pub struct Config {
 
     // Build params
     pub(crate) build_attempts: u16,
+    pub(crate) delay_between_build_attemts: Duration,
```

Comment:
```suggestion
    pub(crate) delay_between_build_attempts: Duration,
```

Reply:
I renamed the setting, thanks for finding it!